packages/landingpage/components/getting-started/layout.module.scss

```diff
@@ -59,3 +59,36 @@
   justify-content: center;
   margin: 5px;
 }
+
+.sandboxImageContainer {
+  position: relative;
+  width: 100%;
+  height: auto;
```

Comment:
`div`s take up 100% width per default and height is redundant as well, I believe